nautobot/extras/models/models.py

```diff
@@ -682,3 +687,49 @@ def log(
             else:
                 log_level = logging.INFO
             logger.log(log_level, str(message))
+
+
+@extras_features("graphql")
+class GraphQLQuery(BaseModel, ChangeLoggedModel):
+    name = models.CharField(max_length=100, unique=True)
+    slug = models.CharField(max_length=100, unique=True)
+    query = models.TextField()
+    variables = models.JSONField(encoder=DjangoJSONEncoder, default=dict, blank=True)
+
+    class Meta:
+        ordering = ("slug",)
```

Comment:
Possibly add:

```python
verbose_name = "GraphQL query"
verbose_name_plural = "GraphQL queries"
```

as I think this will solve the following UI issues:

![image](https://user-images.githubusercontent.com/5603551/119660522-5b3c8300-bdfd-11eb-9515-5d2c453aa24b.png)
and
![image](https://user-images.githubusercontent.com/5603551/119660567-68597200-bdfd-11eb-8e85-1d9f543e46dc.png)